src/components/CalendarBody.tsx

```diff
@@ -218,13 +221,17 @@ function _CalendarBody<T extends ICalendarEventBase>({
   const _renderEvents = React.useCallback(
     (date: dayjs.Dayjs) => {
       if (enableEnrichedEvents) {
-        return (internalEnrichedEventsByDate[date.format(SIMPLE_DATE_FORMAT)] || []).map(
-          _renderMappedEvent,
+        return (
+          <ScrollView horizontal style={styles.horizontalScrollView}>
+            {(internalEnrichedEventsByDate[date.format(SIMPLE_DATE_FORMAT)] || []).map(
+              _renderMappedEvent,
+            )}
+          </ScrollView>
         )
```

Comment:
Please keep the old behavior if let users to choose `enableHorizontalScroll`. In this case they will be wrapped by the `ScrollView` even if a user set `enableHorizontalScroll=false`.